aws/table_aws_batch_job.go

```diff
@@ -0,0 +1,312 @@
+				Hydrate:     getBatchJob,
+				Transform:   transform.FromField("NodeProperties"),
+			},
+			{
+				Name:        "parameters",
+				Description: "Additional parameters that are passed to the job that replace parameter substitution placeholders or override any corresponding parameter defaults from the job definition.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getBatchJob,
+			},
+			{
+				Name:        "platform_capabilities",
+				Description: "The platform capabilities required by the job definition. If no value is specified, it defaults to EC2.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getBatchJob,
+			},
+			{
+				Name:        "propagate_tags",
+				Description: "Specifies whether to propagate the tags from the job or job definition to the corresponding Amazon ECS task. If no value is specified, the tags aren't propagated.",
+				Type:        proto.ColumnType_BOOL,
+				Hydrate:     getBatchJob,
+			},
+			{
+				Name:        "retry_strategy",
+				Description: "The retry strategy to use for this job if an attempt fails.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getBatchJob,
+			},
+			{
+				Name:        "scheduling_priority",
+				Description: "The scheduling policy of the job definition. This only affects jobs in job queues with a fair share policy. Jobs with a higher scheduling priority are scheduled before jobs with a lower scheduling priority.",
+				Type:        proto.ColumnType_INT,
+				Hydrate:     getBatchJob,
+			},
+			{
+				Name:        "share_identifier",
+				Description: "The share identifier for the job.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getBatchJob,
+			},
+			{
+				Name:        "timeout",
+				Description: "The timeout configuration for the job.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getBatchJob,
+			},
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("JobName"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getBatchJob,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("JobArn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listAwsBatchJobs(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := BatchClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_batch_job.listAwsBatchJobs", "connection_error", err)
+		return nil, err
+	}
+	if svc == nil {
+		// Unsupported region, return no data
+		return nil, nil
+	}
+
+	// Limiting the results
+	maxLimit := int32(1000)
```

Comment:
Please update the maxLimit value to `100` as per the [API documentation](https://docs.aws.amazon.com/batch/latest/APIReference/API_ListJobs.html#API_ListJobs_RequestSyntax).